Wurst Client for MC 1.9.X/src/tk/wurst_client/mods/PanicMod.java

```diff
@@ -25,17 +25,27 @@ public void onEnable()
 		wurst.events.add(UpdateListener.class, this);
 	}
 	
+	private ArrayList panicedMods[] = new ArrayList();

```

Comment:
...

On Sat, May 21, 2016 at 3:36 PM, liach <notifications@github.com> wrote:

> In Wurst Client for MC 1.9.X/src/tk/wurst_client/mods/PanicMod.java
> <https://github.com/Wurst-Imperium/Wurst-Client-for-MC-1.9.X/pull/24#discussion_r64139143>
> :
>
> > @@ -25,17 +25,27 @@ public void onEnable()
> >  		wurst.events.add(UpdateListener.class, this);
> >  	}
> >  	
> > +	private ArrayList panicedMods[] = new ArrayList();
>
>
>    1. Missing import java.util.ArrayList
>    2. Missing generics
>    3. remove [] plz
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Wurst-Imperium/Wurst-Client-for-MC-1.9.X/pull/24/files/0216be5689bc12aea02ac6dd2a71e601ae1af2ed#r64139143>
>



-- 
Regards,
Dhruv Gramopadhye
-----
Controll Freak Operator and Everything
dGRAMOP Development
Penetration Testing | Animation and Rendering | Web Development | Mobile
App Development | Embedded Electronics | and Misc Java Development.
------------------------
http://socnet.16mb.com - My most recent and best project - still buggy,
still work in progress. Feel free to create an account
http://dgramop.co - An index page of my projects
http://m.freeplay.url.ph - A neat looking version of the latter link
http://freeplay.url.ph - An index page of random experiments.
http://ppn.hol.es - A currency system that is online that I use to "bribe"
other children. Over 1.5 years old, I already can name a bunch of security
flaws. Try ../ attack when creating an account.
http://m.ppn.hol.es - An even more neat looking version of the above,
scrolling only works on mobile browsers unless you really zoom in. The junk
text is there to enable scrolling.